bevy_kayak_ui/src/render/font/font_mapping.rs

```diff
@@ -62,6 +62,11 @@ impl FontMapping {
         }
     }
 
+    /// Set a default `KayakFont`
+    pub fn set_default(&mut self, handle: Handle<KayakFont>) {
+        self.add("", handle);
```

Comment:
I'd prefer to always use a non-empty string here. It might make sense to have a default font field in the font mapping instead?

Reply:
The only way to make this work is to change `Default` for `FontMapping`.

See 214446621c2eeb95995b6b6b5a9c9ce8eab959e1